modules/guides/pages/transactions.adoc

```diff
@@ -0,0 +1,603 @@
+
+////
+Third, use the query context drop-down menu at the top right of the Query Editor to select `{backtick}travel-sample{backtick}.tenant_agent_00`.
+
+image::transactions-context.png["The query context drop-down menu, with 'travel-sample.tenant_agent_00' selected"]
+////
+
+[source,n1ql]
+----
+include::n1ql:example$transactions/multiple.n1ql[tag=index]
+----
+
+.Transaction
+Now copy the entire sequence below and paste it into the Query Workbench.
+
+[source,n1ql]
+----
+include::n1ql:example$transactions/multiple.n1ql[tag=transaction]
+----
+
+.Result
+[source,json,indent=0]
+----
+include::n1ql:example$transactions/results.jsonc[tag=check-3]
+----
+
+<.> {result}
+--
+
+CBQ Shell::
++
+--
+To execute a transaction containing multiple statements, create the transaction one statement at a time.
+
+Once you have started a transaction, all statements within the cbq shell session are assumed to be part of the same transaction until you rollback or commit the transaction.
+You don't need to set the `txid` parameter.
+
+NOTE: You must be using cbq shell version 2.0 or above to use the automatic transaction ID functionality.
+
+'''
+
+{lead}
+
+.Preparation
+First, specify the transaction settings, as shown in the section <<settings>> above.
+
+Second, create a primary index on `{backtick}travel-sample{backtick}.tenant_agent_00.bookings` so that you can query this collection.
+
+// You must also set the query context to `{backtick}travel-sample{backtick}.tenant_agent_00`.
+
+[source,n1ql]
+----
+include::n1ql:example$transactions/multiple.n1ql[tag=index]
+----
+
+.Transaction
+Now copy the entire sequence below and paste it into the cbq shell.
+
+[source,n1ql]
+----
+include::n1ql:example$transactions/multiple.n1ql[tag=transaction]
+----
+
+.Result
+[source,json,indent=0]
+----
+include::n1ql:example$transactions/results.jsonc[tag=check-3]
+----
+
+<.> {result}
+--
+
+N1QL REST API::
++
+--
+To execute a transaction containing multiple statements, create the transaction one statement at a time.
+
+Once you have started the transaction, you must set the `txid` parameter to specify the transaction to which each subsequent statement belongs.
+
+'''
+
+The following example demonstrates a complete transaction using N1QL.
+Individual N1QL transaction statements are described in the sections below.
+
+.Preparation
+For this example, you will set the query context to `{backtick}travel-sample{backtick}.tenant_agent_00`.
+You must also create a primary index on the `bookings` collection so that you can query it.
+
+[source,console]
+----
+include::n1ql:example$transactions/multiple.sh[tag=index]
+----
+
+.Transaction
+Make the following REST API calls one at a time.
+
+[source,console]
+----
+include::n1ql:example$transactions/multiple.sh[tag=transaction]
+----
+
+<.> {co-txid}
+
+.Result
+[source,json,indent=0]
+----
+include::n1ql:example$transactions/results.jsonc[tag=check-3]
+----
+
+<.> {result}
+--
+====
+
+[[begin]]
+=== Begin a Transaction
```

Comment:
This section should be re-ordered above the first `Multiple Transactions` example, as it makes it much clearer where the `txid` comes from.

Reply:
Could be tricky. I'll try to anticipate the explanation for `txid`

Reply:
Done

Reply:
You know something? Second thoughts, I'm just going to remove all the REST API examples. We haven't mentioned the Query REST API in any of the other how-to guides, the examples are repetitive and hard to reproduce (the user has to substitute the transaction ID every time), and the information is available in the N1QL reference for those who want it.